core/storage/exploration/gae_models.py

```diff
@@ -213,6 +213,9 @@ class ExplorationModel(base_models.VersionedModel):
     # exploration.
     correctness_feedback_enabled = datastore_services.BooleanProperty(
         default=False, indexed=True)
+    # Proto size of the exploration.
```

Comment:
Should we qualify this to indicate it's specifically the size in bytes of Android proto representation of the model?

Reply:
done